src/Contacts/Contact.m

```diff
@@ -86,7 +90,13 @@ - (instancetype)initWithContact:(CNContact *)contact
                 phoneNumberTypeMap[phoneNumber.stringValue] = @(OWSPhoneNumberTypeOtherFAX);
             } else if ([phoneNumberField.label isEqualToString:CNLabelPhoneNumberPager]) {
                 phoneNumberTypeMap[phoneNumber.stringValue] = @(OWSPhoneNumberTypePager);
+            } else if ([phoneNumberField.label isEqualToString:CNLabelOther]) {
+                phoneNumberTypeMap[phoneNumber.stringValue] = @(OWSPhoneNumberTypeOther);
             } else {
+                // TODO this should only be hit if the user has a custom label, which ideally we'd expose rather than
+                // unknown. I think the thing to do is to get rid of OWSPhoneNumberType*, and simply localize the label
+                // in this method - uising the custom label when available.
+                // Maybe that means adding a "label" to PhoneNumber class and parsing the numbers in this class.
```

Comment:
Interesting. We can definitely get rid of the enum. We should localize the system-defined labels though, right?